app/views/users/show.html.haml

```diff
@@ -22,19 +22,17 @@
 
   = render_view_hook "additional_user_fields", f: f
 
-- if can? :edit, @user
-  %ul.inline
+%ul.inline
+  - if current_ability.can?(:switch_to, @user) && @user.active? && !current_facility.cross_facility?
```

Comment:
We also have `Facility#single_facility?` which is the inverse of `cross_facility?`.